database/store.py

```diff
@@ -0,0 +1,174 @@
+        )
+        round_id = application_raw.get("round_id")  # Get round_id if set
+        fund_round = get_round(fund_id, round_id, date_submitted)
+
+        application["id"] = str(uuid.uuid4())  # cant be uuid in restx handler
+        application["status"] = "NOT_STARTED"
+        application["fund_id"] = fund_id
+        # Rename key "name" to "fund_name"
+        application["fund_name"] = application.pop("name")
+        application["round_id"] = fund_round.identifier
+        application["date_submitted"] = date_submitted
+        application["assessment_deadline"] = fund_round.assessment_deadline
+        for question in application.get("questions"):
+            question.update({"status": "NOT STARTED"})
+        return application["id"], application
+
+    def get_status(self, application_id):
+        """
+        Summary:
+            Get status of application and questions
+        Args:
+            application_id: Takes an application_id
+        Returns:
+            Summary of application and assessment status of each question
+        """
+        application_summary = self.applications_index.get(application_id)
+        if application_summary:
+            questions = []
+            for question in self.applications[application_id].get("questions"):
+                questions.append({
+                    "question": question.get("question"),
+                    "status": question.get("status"),
+                })
+            application_summary.update({"questions": questions})
+            return application_summary
+
+    def update_status(
+        self, application_id: str, question_name: str, new_status: str
+    ) -> dict or False:
+        """
+        Summary:
+            Given application_id, update status of question_name to new_status
+        Args:
+            application_id (str): Takes an application_id
+            question_name (str): Takes an question name
+            new_status (str): Takes a status name to be updated
+
+        Returns: status confirmation dict or False.
+        """
+        application = self.applications.get(application_id)
+        for question in application["questions"]:
+            if question["question"] == question_name:
+                question["status"] = new_status
+                return {question_name: new_status}
+        return False
+
+    def search_applications(self, params):
+        """
+        Returns a list of applications matching required params
+        """
+        matching_applications = []
+        datetime_start = params.get("datetime_start")
+        datetime_end = params.get("datetime_end")
+        fund_id = params.get("fund_id")
+        status_only = params.get("status_only")
+        id_contains = params.get("id_contains")
+        order_by = params.get("order_by", "id")
+        order_rev = params.get("order_rev") or False
+        try:
+            order_rev = strtobool(order_rev)
+        except ValueError:
+            pass
```

Comment:
👀 any way to bypass this try except block - sorry these just give me the creeps.

Reply:
`hasattr(params, "order_rev")` ?

Reply:
`params.get("order_rev") == "1"` may be better